src/ui/components/panels/loadout/active_loadout/_ActiveSystemsBlock.vue

```diff
@@ -68,6 +76,18 @@ export default Vue.extend({
     moddedWeapons() {
       return this.mech.MechLoadoutController.ActiveLoadout.Weapons.filter(x => x.Mod)
     },
+    systems(){
+      return this.mech.MechLoadoutController.ActiveLoadout.Systems
+    },
```

Comment:
We would want to define “systems” under the component’s “data”, initializing it with a “mounted” or “created” block. Then whenever we experience a “move” event, we save the component’s “systems” to the mech’s “systems”.